source/exegol-image/my-resources.rst

```diff
@@ -67,6 +67,23 @@ A system exists to easily install arbitrary APT packages in any new exegol conta
 * Importing custom repositories usually requires importing **GPG keys** as well, which can be done by entering trusted GPG keys download URLs in the ``/opt/my-resources/setup/apt/keys.list`` file
 * To install **APT packages** automatically (after updating the repository including the custom ones), just enter a list of package names in the ``/opt/my-resources/setup/apt/packages.list`` file
 
+:code:`bloodhound` (customqueries, config)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+.. seealso::
+    Available from version ``3.1.0`` of any exegol AD image.
+
+A system exists to easily add one or **several** bloodhound customqueries files, or change its configuration file in any new exegol container.
+
+To automatically:
+
+* overwrite the ``~/.config/bloodhound/config.json`` configuration file, simply create the file ``/opt/my-resources/setup/bloodhound/config.json``
+* replace the default exegol customqueries, place one or several valid customqueries files into the folder ``/opt/my-resources/setup/bloodhound/customqueries_replacement/``.
```

Comment:
Add a warning for the replace case that using this method, the user will not receive the future update of the official bloodhound queries from exegol with the next releases

Reply:
I am not sure to see the point of such warning. If the user chooses to replace, he/she does not care about the official release. And if all of a sudden it matters, they just need to move out the files from the `replace` folder and start a new container. A behavior that may apply to most of the my-resources supported tools, doesn't it?
The same applies for the bloodhound config file if they want to revert, and again if the merge option was chosen and they no longer want some parts of the end result.
In my opinion, adding such warning may make the user think that it blocks an update which is not the case as the official file remains in the image. They just need to revert their my-resources customization and start a new container as usual to get the latest update. This bloodhound addition is not special compared to other tools.
In the end, it may add more confusion in the user's mind.

Reply:
@ShutdownRepo what do you think ?

Reply:
Agreeing with @gbe. However, we need to make sure the "my-resources" documentation explains that every made in this features is on the user, and if he/she chooses to replace something, it will most probably replace it